extensions/samples/electron-webpack/README.md

```diff
@@ -0,0 +1,52 @@
+## About
+This is a sample that depicts how to integrate the `msal-node-extensions` library to your [electron](https://www.electronjs.org/) application that has been bundled by [webpack](https://webpack.js.org/).
+
+## Getting started
+To get started using this sample, you need to do the following:
+
+**1. Change the MSAL Electron Configuration** \
+Navigate to the `main.ts` file in the src folder and replace the `<CLIENT_ID>` with your client ID. Follow this [steps](https://docs.microsoft.com/en-us/graph/toolkit/get-started/build-an-electron-app#add-new-application-registration-in-azure-ad-to-get-a-client-id) to generate the client ID from Azure.
+
+```js
+// Initialize the electron authenticator
+const config: MsalElectronConfig = {
+    clientId: '<CLIENT_ID>', // <<== Replace this placeholder with your client ID
+    mainWindow: mainWindow, //BrowserWindow instance that requires auth
+    scopes: [
+    'user.read',
+    'people.read',
+    'user.readbasic.all',
+    'contacts.read',
+    'presence.read.all',
+    'presence.read',
+    'user.read.all',
+    'calendars.read',
+    'Sites.Read.All',
+    'Sites.ReadWrite.All',
+    ],
+    cachePlugin: new PersistenceCachePlugin(persistence),
+};
+```
+
+**2. Run `npm install`** \
+Run `npm install` or your preferred alternative to it, to install the dependencies necessary to run the sample.
+
+```bash
+npm install
+```
+
+**3. Create the DPAPI.node symlink** \
+To allow the `bindings` library to correctly locate the `dpapi.node` file you need to create a symlink to it in the `build` folder on the root of the sample. Run the command below while in the `build` folder on the sample's root to do that.
```

Comment:
```suggestion
To allow the `bindings` library to correctly locate the `dpapi.node` file you need to create a symlink to it in the `dist` folder on the root of the sample. Run the command below while in the `dist` folder on the sample's root to do that.
```

Reply:
I actually removed this section as it was no longer applicable for the sample.